File: ProyectosJAVA/lp2softRRHHController/src/pe/edu/pucp/lp2soft/rrhh/config/DBManager.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pe.edu.pucp.lp2soft.rrhh.config;

public abstract class DBManager {
    public static String urlMSSQL = "jdbc:sqlserver:"
            + "//database-mssql-030521.cmaxickifv0x.us-east-1.rds.amazonaws.com;"
            + "database=lp2;";
    public static String urlMySQL = "jdbc:mysql:"
            + "//database-mysql-030521.cmaxickifv0x.us-east-1.rds.amazonaws.com"
            + ":3306/lp2";

    public static String user = "admin";
    public static String password = "lp220211";
    
}

File: ProyectosNET/LP2soft/LP2softRRHHController/Config/DBManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2softRRHHController.Config
{
    public abstract class DBManager
    {
        public static string cadCon =
            "server=database-mysql-030521.cmaxickifv0x.us-east-1.rds.amazonaws.com;"+
            "user=admin;"+
            "password=lp220211;"+
            "database=lp2;"+
            "port=3306;";
    }
}
